eng/Versions.props

```diff
@@ -148,6 +148,7 @@
     <MicrosoftVisualStudioDiagnosticsMeasurementVersion>17.0.0-preview-1-30928-1112</MicrosoftVisualStudioDiagnosticsMeasurementVersion>
     <MicrosoftVisualStudioDiagnosticsPerformanceProviderVersion>$(MicrosoftVisualStudioShellPackagesVersion)</MicrosoftVisualStudioDiagnosticsPerformanceProviderVersion>
     <MicrosoftVisualStudioSDKEmbedInteropTypesVersion>15.0.36</MicrosoftVisualStudioSDKEmbedInteropTypesVersion>
+    <MicrosoftVisualStudioSearchVersion>17.5.0-preview-1-33102-208</MicrosoftVisualStudioSearchVersion>
```

Comment:
does this need to go into main-vs-deps?

Reply:
It's not super urgent and I think 17.5 P1 is out soon (and I am still testing it)
so I target main now